presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -1034,6 +1034,24 @@ public static void throwOverflowException()
         throw new ArithmeticException("Decimal overflow");
     }
 
+    public static int getInt(Slice decimal, int index)
```

Comment:
please do method move in separate commits. Have this method changed (besides make it them public)?